Shared/Samples/Add dynamic entity layer/AddDynamicEntityLayerView.Model.swift

```diff
@@ -0,0 +1,114 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+extension AddDynamicEntityLayerView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    class Model: ObservableObject {
+        /// A map with a streets basemap style.
+        let map = Map(basemapStyle: .arcGISStreets)
+        
+        /// The data source for the dynamic entity layer.
+        var streamService: ArcGISStreamService = {
+            let streamService = ArcGISStreamService(url: .streamService)
+            
+            let filter = ArcGISStreamServiceFilter()
+            filter.whereClause = "speed > 0"
+            streamService.filter = filter
+            
+            streamService.purgeOptions.maximumDuration = TimeInterval(5 * 60)
+            return streamService
+        }()
+        
+        /// The layer displaying the dynamic entities on the map.
+        var dynamicEntityLayer: DynamicEntityLayer
+        
+        /// A Boolean value indicating whether track lines should be displayed.
+        @Published var showsTrackLine: Bool {
+            didSet {
+                dynamicEntityLayer.trackDisplayProperties.showsTrackLine = showsTrackLine
+            }
+        }
+        
+        /// A Boolean value indicating whether previous observations should be displayed.
+        @Published var showsPreviousObservations: Bool {
+            didSet {
+                dynamicEntityLayer.trackDisplayProperties.showsPreviousObservations = showsPreviousObservations
+            }
+        }
+        
+        /// The maximum number of previous observations to display.
+        @Published var maximumObservations: CGFloat {
+            didSet {
+                dynamicEntityLayer.trackDisplayProperties.maximumObservations = Int(maximumObservations)
+            }
+        }
+        
+        // The maximum observations range.
+        // Used by Slider, which requires CGFloat values.
+        let maxObservationRange = CGFloat(1)...CGFloat(16)
+        
+        /// The stream service connection status.
+        @Published var connectionStatus: String
+        
+        init() {
+            // Create the dynamic entity layer
+            dynamicEntityLayer = DynamicEntityLayer(dataSource: streamService)
+            
+            // Initialize properties from the dynamic entity layer and stream service.
+            showsTrackLine = dynamicEntityLayer.trackDisplayProperties.showsTrackLine
+            showsPreviousObservations = dynamicEntityLayer.trackDisplayProperties.showsPreviousObservations
+            maximumObservations = CGFloat(dynamicEntityLayer.trackDisplayProperties.maximumObservations)
```

Comment:
No need to change, but I feel there is a risk if the server changes its default config `maximumObservations` beyond 1 ~ 16. Slider will show an OOB value.